src/services/config.js

```diff
@@ -156,6 +158,17 @@ class ConfigData {
     // Trim whitespace if given a non-null string.
     return string ? string.trim() : string;
   }
+
+  getCredentialsByProfileID(profileId) {
+    if (this.profiles) {
+      const { profiles } = this;
+      if (profileId && profiles.hasOwnProperty(profileId)) {
+        return { apiKey: profiles[profileId].apiKey, apiSecret: profiles[profileId].apiSecret };
+      }
+      return { apiKey: 'error', apiSecret: 'Profile not found' };
+    }
+    return undefined;
```

Comment:
[Minor] Just checking- Would it be more suited to return `null` in these cases? More like a question about not assigned or marking intentional absence. 

Ref: https://262.ecma-international.org/6.0/#sec-null-value